Update battery documentation (#319)
icub-tech-iit/documentation: docs/robot_power/robot_power_button.md

|:---:|:---:|
| <img src="../assets/fig-icub3-4.png" height="200"> | <img src="../assets/fig-r1-1.png" height="200"> |

## Display Interface

> Lately has been released a new version of the Display UX/UI.

Reviewer: ```suggestion
Lately, we released a new version of the Display UX/UI.
```

> The main screen is as described in the image below for both iCub3, ergoCub and R1:

Reviewer: ```suggestion
The main screen is described in the image below for both iCub3, ergoCub, and R1:
```

<img src="../assets/display-screen-131.png" width="500" height="500">

> Then, if a fault occurs the Display will switch to another screen descring the occurred fault by a string, which summarizes the error type, and the last values of voltage and currents accured at the moment of the fault as shown below (on the right hand side we have errors related to BOARDS/MOTORS line, instead on the left hand side errors related to CPU line):

Reviewer: ```suggestion
If a fault occurs, the Display will switch to a different mode displaying a string that includes the error type and the last values of voltage and currents acuired at the moment of the fault.

Here's a example. On the left, the errors are related to CPU line, whereas on the right, we have errors related to BOARDS/MOTORS line.
```

Reviewer: One more point to address here @MSECode, plus the size of one particular image to reduce as per https://github.com/icub-tech-iit/documentation/pull/319#discussion_r1341498412.
